addOns/ratelimit/src/main/javahelp/org/zaproxy/addon/ratelimit/resources/help/contents/ratelimit.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Rate Limit
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Rate Limit</H1>
+The Rate Limit add-on limits the request rate to hosts or domains to prevent overloading the target and/or being blocked.
+The limit is applied using a matcher against the request host to pick an individual host, an entire domain, or anything
+in between. In the case a host matches multiple rules, the rule with the most restrictive rate is used.
```

Comment:
```suggestion
inbetween. In the case a host matches multiple rules, the rule with the most restrictive rate is used.
```

Reply:
I haven't seen "inbetween" as a single word, dictionary.com didn't think so. Made it "between" because I think "in" isn't necessary.

Reply:
I’m good with that.